wolfssl/wolfcrypt/wc_port.h

```diff
@@ -1180,6 +1180,29 @@ WOLFSSL_ABI WOLFSSL_API int wolfCrypt_Cleanup(void);
     #endif
 #endif
 
+#ifdef WOLF_C99
+    /* use alternate keyword for compatibility with -std=c99 */
+    #define XASM_VOLATILE(a) __asm__ volatile(a)
+#else
+    #define XASM_VOLATILE(a) asm volatile(a)
+#endif
+
+#ifndef WOLFSSL_NO_FENCE
+    #if defined (__i386__) || defined(__x86_64__)
+        #define XFENCE() XASM_VOLATILE("lfence")
+    #elif defined (__arm__) || defined(__aarch64__)
+        #define XFENCE() XASM_VOLATILE("isb")
+    #elif defined(__riscv)
+        #define XFENCE() XASM_VOLATILE("fence")
+    #elif defined(__PPC__)
+        #define XFENCE() XASM_VOLATILE("isync; sync")
+    #else
+        #define XFENCE() do{}while(0)
+    #endif
+#else
+    #define XFENCE() do{}while(0)
```

Comment:
Please have a single all else case after:

```
#ifndef XFENCE
    #define XFENCE() do{}while(0)
#endif
```

Reply:
`XFENCE` is always defined in this `#if` case. Either as the appropriate assembly or as a no-op.